tests/parser/features/decorators/test_nonreentrant.py

```diff
@@ -85,6 +85,104 @@ def unprotected_function(val: String[100], do_callback: bool):
     assert_tx_failed(lambda: reentrant_contract.protected_function2("zzz value", True, transact={}))
 
 
+def test_nonreentrant_decorator_for_default(w3, get_contract, assert_tx_failed):
+    calling_contract_code = """
+@external
+def send_funds(_amount: uint256):
+    # raw_call() is used to overcome gas limit of send()
+    response: Bytes[32] = raw_call(
+        msg.sender,
+        concat(
```

Comment:
Not that it matters much but I think you can use `_abi_encode` here. (In old versions of vyper you were forced to use `concat`)

Reply:
Got it, will make the change so that the test does not appear outdated.